src/app/components/main-layout/main-layout.jsx

```diff
@@ -41,11 +45,16 @@ class MainLayout extends React.PureComponent {
 
     this.warnings = props.topLevelMeta.warnings;
 
+    this.isGroupsSortingEnabled = cookies.get({
+      name: SORT_METHOD_GROUPS_COOKIE_NAME,
+    }) === true.toString();
```

Comment:
I can't say that this is totally wrong, I see why you need this (I mean `true.toString()`). But I have strange feeling that it looks a bit awkward.

If you later choose another value for cookie (so it's not 'true' or 'false'), you need to change it here too or it will break down. I would prefer something like `cookie is set / cookie is undefined` to use good old double negation `!!`.

But as I said, this is just my feeling, you are free to stay with this implementation

Reply:
Thank you! I have fixed it